app/lib/features/chat/providers/notifiers/chat_input_notifier.dart

```diff
@@ -6,11 +6,16 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 class ChatInputNotifier extends StateNotifier<ChatInputState> {
   ChatInputNotifier() : super(const ChatInputState());
 
+  void toggleEditView(bool value) =>
```

Comment:
this is not a toggle, but a set. A toggle would always `value = !value`
(the one below is misnamed, too for that matter..)